### ISSUE TITLE
Media library with playlists and cover grid view?

### ISSUE THREAD

**Issue by AcarBurak** [zaps166/QMPlay2#11]
It may be unpractical to implement or against your design principles, but since QMPlay2 can play both A & W and since there's no such thing already, it would be awesome to have a media library management with playlists and cover grid view.

For audio there's only one such application I'm aware of and it's very intuitive, logical, brilliant: [Cantata](https://github.com/CDrummond/cantata) (mpd client). But mpd is its Achilles heel; all of a sudden or after some system upgrade or other it stops working. So after many years of usage I've given up Cantata. And I believe not many users choose the mpd way with all the hassle, breakage etc.

I've mentioned Cantata also as a reference for what I'm talking about, but if you'd like I would give a step by step explanation about my use case, features request.

Either way, thanks a lot for your good work and efforts.

**Comment by zaps166** [comment]
Something like this: https://nfgk.files.wordpress.com/2013/08/cantata.png?w=613 ?
So I must implement the media library: scanning the Music directory - get artists, titles, albums and store it in database, than display it as albums, titles, artists and show album and song covers...
It is a lot of works, because QMPlay2 uses tree-based playlist. It assumes that user have directories as albums... But anyway I want to improve the playlist, so maybe I'll implement something which will be similar to Cantata's  media library, but it won't be tomorrow :D
Currently I must do something else, so I'm doing mainly bugfixes and minor changes.

**Comment by AcarBurak** [comment]
Yes, your linked view, but there the right pane which is the playque, is unnecessarily wide. In usage it's much narrower and the cover grid on the left is the main window/view.

In fact, the really useful feature is to have covers rather than the metadata thing because any serious A&W collector stores his/her media as you've mentioned, as well-looked after directories of albums. So it's just grid views of covers and ability to filter directory/file names and ability to drag & drop songs to playlists (or doing it by contextual menu).

Anyway, thanks for considering.
